spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/EqlQueryParser.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2022-2023 the original author or authors.
```

Comment:
Nit: Year of inception is the current year across all newly introduced files.

Reply:
Done